src/reducers/orderedReducer.js

```diff
@@ -30,7 +30,7 @@ function modifyDoc(collectionState, action) {
   if (!action.meta.subcollections || action.meta.storeAs) {
     return updateItemInArray(collectionState, action.meta.doc, item =>
       // Merge is used to prevent the removal of existing subcollections
-      mergeObjects(item, action.payload.data),
+      assignObjects(item, action.payload.data),
```

Comment:
I think that this may cause an issue of removing subcollections from the document object. The plan for future versions is to store subcollections separately from the main document, but for now, from what I understand, some folks bank on that data not getting removed.

Reply:
Yeah, when I was looking at it I wondered whether that'd be a problem. Don't think there's really a way around it other than store them separately.